src/core/thread/mesh_forwarder.cpp

```diff
@@ -1840,7 +1840,7 @@ uint16_t MeshForwarder::CalcFrameVersion(const Neighbor *aNeighbor, bool aIePres
     }
 #if OPENTHREAD_FTD && OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
     else if (aNeighbor != nullptr && !Mle::IsActiveRouter(aNeighbor->GetRloc16()) &&
```

Comment:
@abtink Looks like this `!Mle::IsActiveRouter(aNeighbor->GetRloc16())` also can be removed since active router shouldn't be present in `ChildTable`.

Reply:
Yes. Good point. Removed it in new push.